monkey/agent_plugins/payloads/ransomware/config-schema.json

```diff
@@ -26,6 +26,12 @@
             "type": "boolean",
             "description": "If enabled, Infection Monkey will leave a ransom note in the target directory.",
             "default": true
+        },
+        "leave_image": {
```

Comment:
What about, "change_wallpaper"? "leave_image" sounds like it will leave an image file in the target directory.